### ISSUE TITLE
Add keyboard navigation sentences

### ISSUE THREAD

**Issue by luifr10** [Orange-OpenSource/uuv#493]
Desired scenario :
```gherkin
Given I start keyboard navigation on the top of the page # Set keyboard focus at the top of the web page
When I go to next keyboard element # Navigates to the next item selectable by keyboard navigation 
Then the button named "A button" should be focused # Checks that the button named "A button" is focus by keyboard
```

Focusable element is:
  - <a> tag with an href attribute
  - Form control and button (unless the element is disabled)
  - Any element with a tabindex.

- [ ] The following sentences must be created : 
  - `I start keyboard navigation on the top of the page`
  - `I go to next keyboard element`
  - `the element with role {string} and name {string} should be focused`
  - `the button named {string} should be focused`
    - [ ] The sentence `the button named {string} should be focused` must be generated using role based sentence `the element with role {string} and name {string} should be focused`
    - [ ] This sentence must be compatible with all `Within` sentence that can be called before
    - [ ] Role concerned by this sentence :
        - button
        - checkbox
        - combobox
        - form
        - input
        - link
        - listbox
        - listitem
        - option
        - meter
        - progressbar
        - radio
        - radiogroup
        - range
        - search
        - searchbox
        - section
        - select
        - slider
        - spinbutton
        - switch
        - textbox
